### ISSUE TITLE
Large file sync

### ISSUE THREAD

**Issue by BloodyIron** [nextcloud/desktop#545]
My understanding is NC Desktop Sync doesn't have their own version yet (2.5 maybe changing that? not sure). But I've migrated to NC from OC, so I'd prefer to file my ticket here.

When I try to use the desktop sync client to sync really large files over the internet, it often fails. I'm talking like say a 450MB ZIP.

Sync client is 2.4.2 and the server is nextCloud 13.0.2.

Naturally syncing on the LAN does seem to work better/fine, but I haven't tested large file sync very heavily in testing between LAN and internet/WAN.

**Comment by michaelstingl** [comment]
450MB sounds like relatively small file. I'd recommend have a closer look at your server setup, checking logs etc…

More information:
- https://docs.nextcloud.com/desktop/2.3/troubleshooting.html#nextcloud-server-log-file
-  https://docs.nextcloud.com/server/13.0.0/admin_manual/issues/general_troubleshooting.html

**Comment by claell** [comment]
Can this be closed?

**Comment by BloodyIron** [comment]
Within the last few months I actually synced much larger files (52GB worth of video footage) on LAN and it took FOREVER and was very CPU intensive. The issue I observed in that is much worse than the original issue I posted about.

My LAN is fully gigabit, and I can regularly write to the storage at rates of 80MB/s-100MB/s, so storage throughput was really not the issue. Somehow, these files being so big, made the whole process way worse than before, and it really caused a massive load on my laptop (i5-3320m if I remember correctly).

I would say that if you're syncing really really large files on LAN, this is still a problem with the desktop client.
